engines/wintermute/detection.cpp

```diff
@@ -80,6 +81,349 @@ static const ADExtraGuiOptionsMap gameGuiOptions[] = {
 	AD_EXTRA_GUI_OPTIONS_TERMINATOR
 };
 
+struct AchievementDescriptionList {
+	const char *gameId;
+	Common::AchievementsPlatform platform;
+	const char *appId;
+	const Common::AchievementDescription descriptions[64];
+};
+
+#define ACHIEVEMENT_SIMPLE_ENTRY(id, title, comment) {id, false, title, comment}
+#define ACHIEVEMENT_NODESC_ENTRY(id, title) {id, false, title, ""}
+#define ACHIEVEMENT_HIDDEN_ENTRY(id, title) {id, true, title, ""}
+#define ACHIEVEMENTS_LISTEND {0,0,0,0}
+
+static const AchievementDescriptionList achievementDescriptionList[] = {
+	{
```

Comment:
I would move all of these to a separate file, like `achievement_tables.h` or `.cpp`

BTW. keymaps stuff also became way too big, would be nice to move it to a separate file too.

Reply:
Achievements moved at a21623f0df6d288a08044de3302031b6ee18c1fe
Keymaps moved at 0598565d2c841dbfc567dccd32a7d1dea8cd1fe9